cmd/collector/main.go

```diff
@@ -139,6 +140,7 @@ func main() {
 	command.AddCommand(docs.Command(v))
 	command.AddCommand(status.Command(v, ports.CollectorAdminHTTP))
 	command.AddCommand(printconfig.Command(v))
+	featuregate.AddCommand(command)
```

Comment:
please use the same API pattern as other packages. They don't accept `command` argument, they return a `*Command`